migration/v0.20.x-v0.21.x/upgrade-apps.md

```diff
@@ -4,11 +4,19 @@
 
 ## Steps
 
```

Comment:
Can you add a step to run `get-requirements` to update kubectl.

Reply:
done